src/AppM.purs

```diff
@@ -33,8 +40,17 @@ derive newtype instance monadEffectAppM :: MonadEffect AppM
 
 derive newtype instance monadAffAppM :: MonadAff AppM
 
+derive newtype instance monadLoggerAppM :: MonadLogger AppM
+
 instance monadAskAppM :: TypeEquals o Options => MonadAsk o AppM where
   ask = AppM $ asks from
 
 runAppM :: forall a. AppM a -> Options -> Aff a
-runAppM (AppM m) options = runReaderT m options
+runAppM (AppM m) options = runReaderT (runLoggerT m (filterByVerbosity logMessage)) options
+  where
+  filterByVerbosity = case options.verbosity of
+    Verbosity.Info -> minimumLevel Info
+    Verbosity.Debug -> minimumLevel Debug
+    Verbosity.Silent -> const $ const $ pure unit
+
+  logMessage message = liftEffect $ prettyFormatter message >>= Console.log
```

Comment:
You should be able to write

```purescript
logMessage = prettyFormatter >=> log
```

If you use the `log` function from `Effect.Class.Console`, since its type is `MonadEffect m => String -> m Unit`, not `String -> Effect Unit` 😄 

Which one do you prefer?

Reply:
Oh yeah, of course the one from `Effect.Class.Console`!